### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

## Related Issue

- Fixes <issue_link>

## Motivation and Context

Typos can result in tests failing to test what they intend to test. Visible typos show a lack of polish. Typos can result in poor translations. And typos can mean that when searching for an identifier, things will be missed.

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...
The misspellings have been reported at https://github.com/jsoref/owncloud-ocis/commit/213adeadef66e6f339080a5e2cd521dcdd3da75b#commitcomment-56646320
The action reports that the changes in this PR would make it happy: https://github.com/jsoref/owncloud-ocis/commit/ba53e1b147c5a47b9226ab15bd41c2d9ba025e1f

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: accounts/pkg/proto/v0/accounts.pb.micro_test.go
Comment (jsoref) [inline accounts/pkg/proto/v0/accounts.pb.micro_test.go:620]: This appears to be the name of the library...

File: deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json
Comment (jsoref) [inline deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json:1454]: scary

File: docs/ocis/adr/0006-service-discovery.md
Comment (jsoref) [inline docs/ocis/adr/0006-service-discovery.md:40]: Not sure about this. Mostly concerned about `rega`

File: glauth/templates/CONFIGURATION.tmpl
Comment (jsoref) [inline glauth/templates/CONFIGURATION.tmpl:50]: The vast majority of corrections were picked automatically by Google Sheets. This pair wasn't.

File: idp/ui/src/errors/index.js
Comment (jsoref) [inline idp/ui/src/errors/index.js:6]: scary

File: proxy/pkg/proxy/policy/selector_test.go
Comment (jsoref) [inline proxy/pkg/proxy/policy/selector_test.go:144]: As this is a test, and as I've already tripped on one intentional typo testcase, it wouldn't shock me if this is wrong.

If it's wrong, it'd help if the test case clearly spelled out that it was an intentional typo.

File: settings/ui/components/SettingsApp.vue
Comment (jsoref) [inline settings/ui/components/SettingsApp.vue:30]: ??

partition?

File: storage/pkg/command/frontend.go
Comment (jsoref) [inline storage/pkg/command/frontend.go:33]: cli api break?

File: storage/pkg/flagset/sharingsql.go
Comment (jsoref) [inline storage/pkg/flagset/sharingsql.go:9]: probably?

**Review by jsoref — COMMENTED** [review]
As usual, I can drop things as requested, if there's an upstream, let me know. But note that I probably can't do anything to any of these PRs until October.

**Comment by jsoref** [comment]
@pascalwengerter asked for this: https://github.com/owncloud/web/pull/5815#pullrequestreview-757185301 but I was already planning on it, it just got missed when I was working on my initial set.

Oddly, there doesn't appear to be a license-bot, not sure why.

File: docs/ocis/adr/0006-service-discovery.md
Comment (pascalwengerter) [reply docs/ocis/adr/0006-service-discovery.md:40]: Most likely a reference to github.com/cs3org/reva/ and a fix we can apply ourselves :)

**Comment by phil-davis** [comment]
Note: @jsoref has said that he is happy for us to adjust this PR, rebase etc to get it how we need.

(GitHub probably lets us push the branch in his fork)

File: settings/ui/components/SettingsApp.vue
Comment (pascalwengerter) [reply settings/ui/components/SettingsApp.vue:30]: Should be fine, this only applies a styling variation on the respective element ;) `variation` is the right prop so thanks for fixing!

**Comment by pascalwengerter** [comment]
> Note: @jsoref has said that he is happy for us to adjust this PR, rebase etc to get it how we need.
> 
> (GitHub probably lets us push to the the branch in his fork)

Needs a proper review and a changelog item _if_ we have breaking CLI changes. This could be done after merging/before the next release or by changing the PR target branch to one we maintainers have access to

Whoever merges this please use squash-on-merge if available in this repo

And thanks @jsoref already!

File: storage/pkg/command/frontend.go
Comment (butonic) [reply storage/pkg/command/frontend.go:33]: better now than after GA

File: accounts/pkg/proto/v0/accounts.pb.micro_test.go
Comment (pascalwengerter) [reply accounts/pkg/proto/v0/accounts.pb.micro_test.go:620]: Yes, but your fix ist still valid since we're using it in plural throughout the codebase

**Review by pascalwengerter — COMMENTED** [review]
Would love to get a feedback by e.g. @wkloucek on changes in `deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json` but for the rest should be fine to merge from my POV (plus changelog item regarding CLI fixes which are kinda breaking)

File: deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json
Comment (wkloucek) [reply deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json:1454]: there are many tickets on this, eg. https://issues.redhat.com/browse/KEYCLOAK-16928. So this looks good to me :+1:

File: proxy/pkg/proxy/policy/selector_test.go
Comment (wkloucek) [reply proxy/pkg/proxy/policy/selector_test.go:144]: looks good :-)

**Comment by wkloucek** [comment]
> Would love to get a feedback by e.g. @wkloucek on changes in `deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json` but for the rest should be fine to merge from my POV (plus changelog item regarding CLI fixes which are kinda breaking)

Keycloak changes are ok, since they had a typo on their own, but they support also the non-typo version of it :-D

When we have a changelog with a breaking notice because of the CLI change I'm happy to merge

**Comment by phil-davis** [comment]
Who is going to write the changelog?

@jsoref is off-line for a while now.

**Comment by pascalwengerter** [comment]
> Who is going to write the changelog?
> 
> @jsoref is off-line for a while now.

@wkloucek @phil-davis let's merge this and I'll open another PR providing the changelog directly afterwards, needs approval by one of you

File: docs/ocis/adr/0006-service-discovery.md
Comment (phil-davis) [reply docs/ocis/adr/0006-service-discovery.md:40]: fixed s/rega/reva

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Approved and squashed. Ready for merge when CI finishes.

**Pull request merged**
